shared/constants/permissions.ts

```diff
@@ -29,4 +29,45 @@ export const ConnectionPermission = Object.freeze({
   connection_permission: 'connection_permission',
 });
 
+// This configuration specifies permission weight thresholds used to determine which
+// permissions to show or hide on certain Snap-related flows (Install, Update, etc.)
+export const PermissionWeightThreshold = Object.freeze({
+  snapInstall: 3 as const,
+  snapUpdateApprovedPermissions: 2 as const,
```

Comment:
```suggestion
  snapUpdateApprovedPermissions: 3 as const,
```

I thought we agreed this threshold should be `3` too 😄

Reply:
It is following the logic proposed by Erik. The thing is that I use the same function which was initially made for Snap install, so following the logic proposed I started with having `<=` in the condition, like this:
```javascript
(permission) => permission.weight <= (weightThreshold ?? Infinity)
```
So in order to respect the logic for both, install and update flows, I need to make number adjustment on one side. Or we can increase the numbers and use strictly `<` in the condition?

![Screenshot 2024-08-14 at 11 23 18](https://github.com/user-attachments/assets/47158170-87af-4750-ad32-5b68ee75bc02)

Let me know which way you prefer, I can update.

Reply:
I think it is simplest if we just use `3` for both for now. I think that is what Erik meant as well, even if the comment is confusing 😅

Reply:
Updated as requested.